pynecone/app.py

```diff
@@ -65,6 +66,12 @@ class App(Base):
     # List of event handlers to trigger when a page loads.
     load_events: Dict[str, List[EventHandler]] = {}
 
+    # The component to render if there is a connection error to the server.
+    connect_error_component: Optional[Component] = None
+
+    # Whether to render component if server cant be reached.
```

Comment:
Maybe we can combine this with the above? If it is set to `None` we don't check the connection since there's nothing to display